homeassistant/components/zha/core/channels/closures.py

```diff
@@ -62,4 +62,72 @@ class Shade(ZigbeeChannel):
 class WindowCovering(ZigbeeChannel):
     """Window channel."""
 
-    pass
+    _value_attribute = 8
+    REPORT_CONFIG = (
+        {"attr": "current_position_lift_percentage", "config": REPORT_CONFIG_IMMEDIATE},
+    )
+
+    async def async_open(self) -> None:
+        """Commands the covering to open."""
+        await self.device.issue_cluster_command(
```

Comment:
Why? 
Do it from the Entity, same as light does https://github.com/home-assistant/home-assistant/blob/d25aa1f1830f1d3fad42dde3f9bdfae2ad98b7f2/homeassistant/components/zha/light.py#L230

e.g. `result = await self._cover_chanel.up_open()`